src/main/java/org/jabref/model/entry/field/Field.java

```diff
@@ -16,8 +16,12 @@ public interface Field {
     /**
      * @return A version of the field name more suitable for display
      */
+//    default String getDisplayName() {
+//        return StringUtil.capitalizeFirst(getName());
+//    }
+
```

Comment:
Please remove artifacts instead of commenting it. This is what a VCS like git is made for.